libr/anal/esil.c

```diff
@@ -74,6 +74,11 @@ static bool popRN(RAnalEsil *esil, ut64 *n) {
 
 /* R_ANAL_ESIL API */
 
+static void esil_ops_free(HtPPKv *kv) {
+	free (kv->key);
```

Comment:
Check if kv is null

Reply:
Current usage is correct but for consistency eith all the free methods should work when passing null as arg

Reply:
Actually I think there is no ht_ free method so far that accepts NULL. In any case, Ht will never call the free method on a NULL object (at least it shouldn't, unless there are bugs and in that case it would be better to catch them anyway, but I don't think there are as I haven't seen those kind of issues so far).